msu/utils/type_checkers.nut

```diff
@@ -8,6 +8,19 @@
 	::MSU.requireTypeArray("integer", vargv);
 }
 
+::MSU.requireUnsigned <- function( ... )
+{
+	::MSU.requireTypeArray("integer", vargv);
```

Comment:
I will say this use only seemed good back then.
But in reality this means that failing an unsigned integer check will print a "failed an integer check" message half the time. And that is not fully correct.

Ideally we would rewrite the integer check logic in this check